ERCS/erc-_____.md

```diff
@@ -0,0 +1,342 @@
+---
+eip: ____
+title: "Contactless Crypto Payment Standard"
+description: Standard for initializing contactless payment transactions from EVM wallets
+author: Amhed Herrera, Justin Lee, Arjun Dureja
+discussions-to: 
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-10-25
+requires: 20, 681, 712
+---
+
+## Abstract
+This ERC defines a standard for contactless payment transactions that can allow for interoperable customer to merchant onchain payments, regardless of the wallets the customer and merchant are using.
+
+## Motivation
+Currently there is no standard mechanism in crypto to do contactless payment transactions via NFC. This ERC defines a standardized way of exchanging payment information between merchants and customers, or peer-to-peer individuals, so that efficient checkout can occur.
+
+## Specification
+
+Comprised of three parts:
+- A new web3Provider method called `requestContactlessPayment`
```

Comment:
Is web3Provider an 1193 provider or something else? Also I think these generally have prefixes like eth_ or wallet_

Reply:
Yup, it's the Ethereum Provider JavaScript API. Will update